auth/src/test/java/com/firebase/ui/auth/test_helpers/FacebookProviderShadow.java

```diff
@@ -44,8 +41,6 @@ public FacebookProviderShadow() {
         }
     }
 
-    public void __constructor__(Activity activity, List<String> scopes) {}
```

Comment:
From what I've read (see [Shadowing Constructors](http://robolectric.org/extending/) at the bottom of the page), this would be used to collect variables from the constructor, but it was empty. If you believe it serves a purpose, I will add it back.